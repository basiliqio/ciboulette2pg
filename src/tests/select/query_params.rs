use super::*;

ciboulette_query_test_multi!(test_select);
ciboulette_query_test_single!(test_select);
