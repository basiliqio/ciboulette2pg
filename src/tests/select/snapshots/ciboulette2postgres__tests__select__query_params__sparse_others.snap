---
source: src/tests/select/query_params.rs
expression: value

---
[
  {
    "id": "[uuid]",
    "type_": "articles",
    "data": {
      "title": "How to dead"
    }
  },
  {
    "id": "[uuid]",
    "type_": "articles",
    "data": {
      "title": "Oh my g**"
    }
  },
  {
    "id": "[uuid]",
    "type_": "people-article",
    "data": {
      "article_id": "[uuid]",
      "people_id": "[uuid]"
    }
  },
  {
    "id": "[uuid]",
    "type_": "people-article",
    "data": {
      "article_id": "[uuid]",
      "people_id": "[uuid]"
    }
  },
  {
    "id": "[uuid]",
    "type_": "peoples",
    "data": {
      "age": 22,
      "first-name": "Francis",
      "gender": "M",
      "last-name": "Le Roy",
      "twitter": null
    }
  }
]
